.github/workflows/configure.cmd

```diff
@@ -36,7 +36,7 @@ cmake                                                ^
   -G"Visual Studio 16 2019"                          ^
   -H"%KRATOS_SOURCE%"                                ^
   -B"%KRATOS_BUILD%\%KRATOS_BUILD_TYPE%"             ^
-  -DBOOST_ROOT="%BOOST_ROOT_1_72_0%"                 ^
+  -DBOOST_ROOT="%TEMP\boost%"                        ^
```

Comment:
this seems to be the remaining issue
@roigcarlo  @oberbichler I think this should be easy to solve, I just don't know how this is done in Win...

Reply:
You should put the path direclty here right? Right now is trying to find an env variable with name "TEMP\boost". 

Did not test this, just a guess:
```suggestion
  -DBOOST_ROOT="$env:TEMP\boost"                        ^
```

Reply:
no clue, but thx :)

Reply:
Actually, in the issue, below the code snippet,  they mention:

`The above code snippet will install Boost to the C:\hostedtoolcache\windows\Boost\1.72.0\x86_64 folder.`

So maybe thats the path we should use :D

Reply:
Can you make the change?
Just to avoid misunderstandings 

(btw the INSTALL it we only download it. Not sure if that’s important)

Reply:
Thx @marcnunezc !